internal/broker/bind_create_test.go

```diff
@@ -234,7 +192,7 @@ func TestCreateBindingEndpoint(t *testing.T) {
 
 		//// verify connectivity using kubeconfig from the generated binding
 		newClient := kubeconfigClient(t, binding.Credentials.(string))
-		_, err = newClient.CoreV1().Secrets("default").Get(context.Background(), "secret-to-check", v1.GetOptions{})
+		_, err = newClient.CoreV1().Secrets("kyma-system").Get(context.Background(), "secret-to-check", v1.GetOptions{})
```

Comment:
I propose to keep default namespace since the kubeconfig should allow accessing all namespaces.